opflexagent/gbp_ovs_agent.py

```diff
@@ -35,7 +35,6 @@
 from opflexagent import as_metadata_manager
 from opflexagent import constants as ofcst
 from opflexagent import rpc
-from opflexagent import snat_iptables_manager
```

Comment:
I thought I already fixed the root helper issue on Kilo, see https://github.com/noironetworks/python-opflex-agent/commit/9c5a7d9a2842b450b09bc15d3883ac7ea8d4afb5